tools/shared/src/main/scala/org/scalajs/core/tools/javascript/JSDesugaring.scala

```diff
@@ -800,7 +801,7 @@ object JSDesugaring {
      *  Return
      */
     def pushLhsInto(lhs: Tree, rhs: Tree)(implicit env: Env): js.Tree = {
-      implicit val rhsPos = rhs.pos
+      implicit val pos = rhs.pos
```

Comment:
Note: this change is so that the `implicit val pos = value.pos` down there shadows this one. Otherwise this causes ambiguous implicits. I don't know why it's ever been named `rhsPos` anyway ...